app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
coding convention
because, this method have not body, you can write it one line
`def home; end`

Reply:
done